apps/vs-code-react/src/app/overview/app.tsx

```diff
@@ -11,13 +11,25 @@ import { useCallback, useContext, useMemo } from 'react';
 import { useInfiniteQuery, useMutation } from 'react-query';
 import { useSelector } from 'react-redux';
 import invariant from 'tiny-invariant';
+import { useIntl } from 'react-intl';
+
 export interface CallbackInfo {
   method?: string;
   value: string;
 }
 export const OverviewApp = () => {
   const workflowState = useSelector((state: RootState) => state.workflow);
   const vscode = useContext(VSCodeContext);
+  const { isWorkflowRuntimeRunning } = workflowState;
+  const intl = useIntl();
+
+  const intlText = {
+    DEBUG_PROJECT_ERROR: intl.formatMessage({
+      defaultMessage: 'Please start the project by pressing F5 or run it through the Run and Debug view',
```

Comment:
@ecfan could you please review this error message?

Reply:
@ccastrotrejo: Apologies, am on vacation until April 8.

Reply:
No problem at all, I can go back later and change the wording. Thanks!